### PR TITLE
feat(state): add lynis.config state

### PR DESCRIPTION
**Pull request by n-rodriguez**
### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [x] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Documentation checklist

- [x] Updated the `README` (e.g. `Available states`).
- [x] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [x] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [x] Updated the relevant test pillar.

### CODE REVIEW

**Comment by n-rodriguez** [comment]
@myii ready https://travis-ci.com/github/saltstack-formulas/lynis-formula :rocket:

File: lynis/libtofs.jinja [hunk 1]
```diff
@@ -0,0 +1,112 @@
+{%- macro files_switch(source_files,
```
Comment (myii) [inline lynis/libtofs.jinja]: This is an older version of `libtofs.jinja`.  The latest version is available here:

* https://github.com/saltstack-formulas/template-formula/blob/master/TEMPLATE/libtofs.jinja

**Review by myii — CHANGES_REQUESTED** [review]
All fine otherwise, just need the latest version of `libtofs.jinja`, thanks.

---

Checked using https://github.com/myii/ssf-formula/pull/260.

File: lynis/libtofs.jinja (hunk 1 above)
Comment (n-rodriguez) [reply lynis/libtofs.jinja]: fixed!

**Review by myii — APPROVED** [review]

**Comment by myii** [comment]
Thanks, @n-rodriguez -- merged.

**Comment by n-rodriguez** [comment]
@myii Thank you! :+1:

**Pull request merged**